.travis.yml

```diff
@@ -153,25 +150,28 @@ before_install:
            brew unlink gcc
            brew install gcc@8
            brew link gcc@8
+           brew uninstall numpy
```

Comment:
I'd suggest to remove this again. The log shows that brew refuses to uninstall numpy (1.19.4 at present), because it is a dependency.

Reply:
Sure!